routes/web.php

```diff
@@ -21,6 +22,11 @@
     return view('welcome');
 });
 
+// Contact Routes
+Route::get('/contact', [App\Http\Controllers\ContactController::class, 'index']);
+Route::post('index', [App\Http\Controllers\ContactController::class, 'store'])->name('store');
```

Comment:
Geef deze functies een duidelijke unieke naam voor wat ze doen, dus in dit geval contact_store en contact_index.